src/App.js

```diff
@@ -25,6 +25,7 @@ function App() {
       ["Aathish04", "Asia/Kolkata"],
       ["kilacoda", "Asia/Kolkata"],
       ["naveen521kk","Asia/Kolkata"],
+      ["Darylgolden", "Singapore"]
```

Comment:
is `Singapore` right here? Can you confirm? (see how other's timezone are formatted.)

Reply:
Assuming you're using the [tz database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), yes

Reply:
Oh nvm, it's deprecated

Reply:
Fixed